works.yaml

```diff
@@ -5478,22 +5447,6 @@ OpenBazaar Client:
   - https://github.com/OpenBazaar/OpenBazaar-Client
   description:
   - The OpenBazaar client is a front-end Electron application for talking with the OpenBazaar-Server daemon.
-OpenBazaar Connector:
```

Comment:
Please re-add this project with the following as its homepage and source of other information:

https://github.com/STRML/openbazaar-connector